app/src/test/kotlin/com/wire/android/util/DeepLinkProcessorTest.kt

```diff
@@ -79,6 +79,17 @@ class DeepLinkProcessorTest {
         )
     }
 
+    @Test
+    fun `given a incoming call deeplink, returns IncomingCall with conversationId`() {
+        generateIncomingCallDeepLink()
```

Comment:
This is not really generating anything as I'm not using the result either, but rather setting up all the stubbing. Shall we rename it to `setupWithIncomingCallDeepLink` or something like that?

Reply:
make sense, renamed all such funs in test